### ISSUE TITLE
A11Y: buttons on main screen have no labels

### ISSUE THREAD

**Issue by paul-geoghegan** [mijorus/smile#152]
I am using a screen reader on Ubunt called Orca. You can turn it on by pressing super+alt+s. There are many unlabeled buttons in the app for example the main screen has 12. You can find them by:
1. Open the app
2. Turn on Orca (super+alt+s),
3. pressing shift+tab which moves your focus backwards (tab goes forwards),
4. Keep pressing shift and tab and you will discover the 12 unlabeled buttons one after another.
I would expect these to be labeled so that blind people like me can propperly understand what they are for.
Let me know if you have any questions or if I can help.
